cron_update_public_tribe.sh

```diff
@@ -0,0 +1,10 @@
+#! /bin/bash
```

Comment:
So this will be the script that cron job will run weekly?

Reply:
Yes, exactly.